lib/ansible/modules/system/user.py

```diff
@@ -744,17 +760,21 @@ def modify_user_usermod(self):
                 if current_expires < 0 or current_expire_date[:3] != self.expires[:3]:
                     cmd.append('-e')
                     cmd.append(time.strftime(self.DATE_FORMAT, self.expires))
+                    self.changes['expires'] = (current_expires, self.expires)
```

Comment:
Format the date string here so it's more human readable in the diff:
```suggestion
                    self.changes['expires'] = (current_expires, time.strftime(self.DATE_FORMAT, self.expires))
```

Reply:
In that case I think I should also format `current_expires` the same way.